import-service/import/import-file-parser.ts

```diff
@@ -0,0 +1,36 @@
+import * as AWS from "aws-sdk";
+import * as AWSLambda from "aws-lambda";
+const csv = require("csv-parser");
+const BUCKET = "app-import-storage";
+
+export const importFileParser = (event: AWSLambda.S3Handler) => {
+  const s3 = new AWS.S3({ region: "eu-west-1" });
+  console.log("START parse data");
+  event.Records.forEach((record) => {
+    const s3Stream = s3.getObject({ Bucket: BUCKET, Key: record.s3.object.key }).createReadStream();
+
+    s3Stream
+      .pipe(csv())
+      .on("data", (data) => {
+        console.log("DATA STREAM: ", data);
+      })
+      .on("end", async () => {
+        await s3
+          .copyObject({
+            Bucket: BUCKET,
+            CopySource: `${BUCKET}/${record.s3.object.key}`,
+            Key: record.s3.object.key.replace("uploaded", "parsed"),
+          })
+          .promise();
+        console.log(`COPY from ${BUCKET}/${record.s3.object.key}`);
+
+        await s3
+          .deleteObject({
+            Bucket: BUCKET,
+            Key: record.s3.object.key,
+          })
+          .promise();
+        console.log(`DELETE - ${BUCKET}/${record.s3.object.key}`);
+      });
+  });
```

Comment:
You don't have any error handling here. As for me it's reasonable at least see error stack trace in CloudWatch. What do you think about it?